### PR TITLE
Change layout depending on available space

### PR DESCRIPTION
**Pull request by crswll**
Adding the ability to split the panes horizontally or vertically based on the users preference.

### CODE REVIEW

**Comment by anteriovieira** [comment]
Hi @crswll , you can provide a gif to demonstrate the behavior?

**Comment by crswll** [comment]
Sure.

**Comment by Akryum** [comment]
Thanks for your PR @crswll :+1: 
What if we switch the layout automatically depending on the width of the devtools?

**Comment by crswll** [comment]
No problem! I'm happy to help.

I didn't do it with media queries out of personal preference. There's was too many times in devtools I'd want a wide view but then it'll turn into two columns instead of one before i changed the "Panel Layout" setting.

I wonder if I can read that setting and honor it and use media queries if it's set to auto.

Feel all right about that?

**Comment by Akryum** [comment]
Why not, we can try a 3-state button with h/v/auto then.

**Comment by crswll** [comment]
Oh no I meant just getting the setting directly from Chrome devtools so I'd remove the button all together and change a class based on that setting.

**Comment by Akryum** [comment]
Oh okay 👍

**Comment by crswll** [comment]
I gave it a quick look and a long Google but I don't think that setting is exposed like the theme unfortunately.

Any of you know if this setting is exposed?

screen shot 2017-12-24 at 9 51 08 pm

`chrome.devtools.panels.themeName` exists but `chrome.devtools.panels.panelLayout` or the like does not seem to.

I'll go with the 3-state button if there's no luck here.

**Comment by crswll** [comment]
I'm finally getting around to this and not digging a three state button. I'm just going to use a media query and have it work "auto" until I figure out how to read that setting and honor that.

**Comment by crswll** [comment]
Ok, now 2 columns when we have over 685px, which is the breakpoint Chrome devtools seems to use.

File: src/devtools/App.vue [hunk 1]
```diff
@@ -31,11 +31,10 @@
       <span class="pane-name">Events</span>
       <span class="event-count" v-if="newEventCount > 0">{{ newEventCount }}</span>
     </a>
-    <a class="button refresh"
+    <a class="button refresh icon-only"
```
Comment (Akryum) [inline src/devtools/App.vue]: Why setting icon only here?

Comment (crswll) [reply src/devtools/App.vue]: I removed the button that toggles the layout and that had it as well. I can change this to a specific class and remove this more utilitarian class.

Comment (Akryum) [reply src/devtools/App.vue]: I mean, we can make the Refresh button normal (by removing this class) now that you removed the other icon.

Comment (crswll) [reply src/devtools/App.vue]: All right. It's been removed along with the extra class.

**Comment by crswll** [comment]
Quick shot of it working.

**Pull request merged**